app/views/provider_interface/application_choices/show.html.erb

```diff
@@ -49,9 +49,9 @@
 
     <%= render SummaryListComponent.new(rows: [
       { key: 'Provider', value: @application_choice.provider.name_and_code },
-      { key: 'Course', value: @application_choice.offered_course.name_and_code },
+      { key: 'Course', value: @application_choice.course.name_and_code },
```

Comment:
Guessing the intention here is to display the original choice? It's not super clear, but guessing this will be redesigned soon enough.

Reply:
Yeah, I thought it was confusing to mutate this as we have a separate `offer` section